### ISSUE TITLE
Can't see which block is hovered on "High Contrast" mode

### ISSUE THREAD

**Issue by Helen09** [Microsoft/pxt-adafruit#645]
**Repro steps:**
• Navigate to https://makecode.adafruit.com/beta
• Click "LIGHT" and then hover on each block
• Click "setting" and then click "High Contrast On" and hover on block again

**Expect result:**
When hover on one block, this block should have a color shade change or other
![5](https://user-images.githubusercontent.com/23466737/36627698-967b8edc-193e-11e8-8750-3ae180fa5d73.gif)

**Actual result:**
After clicking "High Contrast On" and then hover on blocks, there is no color change, so user can't see which block is selected
![6](https://user-images.githubusercontent.com/23466737/36627696-91b1a5bc-193e-11e8-9f3b-cb014563010e.gif)

**Comment by samelhusseini** [comment]
Right, tron look again.

**Comment by pelikhan** [comment]
Time to rip it out.

________________________________
